src/main/java/com/facebook/openwifirrm/optimizers/channel/RandomChannelInitializer.java

```diff
@@ -46,6 +50,27 @@ public RandomChannelInitializer(
 		super(model, zone, deviceDataManager);
 	}
 
+	/** Constructor (allows setting different channel per AP) */
+	public RandomChannelInitializer(
+		DataModel model, String zone, DeviceDataManager deviceDataManager, boolean setDifferentChannelsPerAp
+	) {
+		this(model, zone, deviceDataManager);
```

Comment:
I am pretty sure you wrote this idiom backwards (based on what I've seen). It should go this way, which looks cleaner:
```java
public class X {
	private final int a, b;
	public X() {
		this(1);
	}
	public X(int a) {
		this(a, 2);
	}
	public X(int a, int b) {
		this.a = a;
		this.b = b;
	}
}
```

Also please fix this in the other class as well.